iree/hal/vulkan/serializing_command_queue.h

```diff
@@ -72,14 +72,15 @@ class SerializingCommandQueue final : public CommandQueue {
   void AbortQueueSubmission();
 
  private:
+  struct PendingBatch {
```

Comment:
I think we can just embed this struct inside `FencedSubmssion`. But it's a small issue so your call. :)

Reply:
Modeled after iree/hal/host/serial/serial_submission_queue: https://github.com/google/iree/blob/a672e635d09960a6acb270b79af850dfd4103637/iree/hal/host/serial/serial_submission_queue.h#L73-L81